05/Validation.js

```diff
@@ -0,0 +1,52 @@
+const formFields = [
+    {name: "names", label: "Name and surname", pattern: /^[^\s]+( [^\s]+)+$/,  required: true},
+    {name: "email", label: "Mail", pattern: /^[a-zA-Z0-9.!#$%&'*+/=?^_`{|}~-]+@[a-zA-Z0-9-]+(?:\.[a-zA-Z0-9-]+)*$/,  required: true},
+    {name: "phone", label:"Phone", pattern: /^\+\d{1,12}$/, required: true },
```

Comment:
Dość mocno nieoczywiste było dla mnie że muszę wpisać + na początku, aby format był poprawny.
Jeśli mamy takie niestandardowe rozwiązania to należy o tym napisać w formularzu lub w błędzie :P